mavo-sort.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+
+Mavo.Plugins.register("sort", {});
+
+Mavo.Functions.sort = function(array, ...properties) {
+	var arrayCopy = array.slice();
+	var splitChar = ".";
```

Comment:
Why is this a variable? Is there any chance it might change? If not, I don't think assigning it to a variable improves readability. Despite what some people claim, literals are not the devil :)